fdbserver/Status.h

```diff
@@ -27,14 +27,7 @@
 #include "fdbserver/MasterInterface.h"
 #include "fdbclient/ClusterInterface.h"
 
-typedef Standalone<VectorRef<StringRef>> ProcessIssues;
-typedef std::map<NetworkAddress, std::pair<ProcessIssues, UID>> ProcessIssuesMap;
```

Comment:
I kind of liked these typedefs, as the type `std::vector<std::pair<NetworkAddress, Standalone<VectorRef<StringRef>>>>` is a bit unwieldy. What do you think about reintroducing `ProcessIssues`, so that we end up with `std::vector<std::pair<NetworkAddress, ProcessIssues>>`? Or I suppose we could create a little struct:

```
struct ProcessIssues {
    NetworkAddress address;
    Standalone<VectorRef<StringRef>> issues;
};
```

which would give us `std::vector<ProcessIssues>` and make the members accessible by more descriptive names.